### PR TITLE
Add support for Laravel 7 + PHP 7.4

### PR DESCRIPTION
**Pull request by Okipa**
Hi there, here is my implementation proposal for :

* Laravel 7 support addition.
* PHP 7.4 support addition.
* PHP 7.0 support drop.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,16 +11,16 @@
         }
     ],
     "require": {
-        "php": ">=7.0",
+        "php": "^7.1",
```
Comment (atymic) [inline composer.json:14]:
```suggestion
        "php": "^7.2",
```

File: composer.json [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
         "guzzlehttp/guzzle": "~6.0",
-        "illuminate/notifications": "~5.5 || ~6.0",
-        "illuminate/support": "~5.5 || ~6.0"
+        "illuminate/notifications": "~5.5||^6.0||^7.0",
+        "illuminate/support": "~5.8.0||^6.0||^7.0"
```
Comment (atymic) [inline composer.json:17]: This should match the `illuminate/notifications` package

File: composer.json [hunk 3]
```diff
@@ -11,16 +11,16 @@
...
     },
     "require-dev": {
-        "mockery/mockery": "^0.9.5",
-        "phpunit/phpunit": "~7.0 || ~8.0",
-        "orchestra/testbench": "^3.5.0 || ^4.0",
-        "orchestra/database": "^3.5.0 || ^4.0"
+        "mockery/mockery": "^0.9.5||^1.0",
+        "phpunit/phpunit": "^7.0||^8.0",
```
Comment (atymic) [inline composer.json:21]: Just drop 7.0 :)

File: composer.json [hunk 4]
```diff
@@ -11,16 +11,16 @@
...
-        "illuminate/support": "~5.5 || ~6.0"
+        "illuminate/notifications": "~5.5||^6.0||^7.0",
+        "illuminate/support": "~5.8.0||^6.0||^7.0"
     },
     "require-dev": {
-        "mockery/mockery": "^0.9.5",
-        "phpunit/phpunit": "~7.0 || ~8.0",
-        "orchestra/testbench": "^3.5.0 || ^4.0",
-        "orchestra/database": "^3.5.0 || ^4.0"
+        "mockery/mockery": "^0.9.5||^1.0",
```
Comment (atymic) [inline composer.json:20]: Also drop 0.x here

**Review by atymic — CHANGES_REQUESTED** [review]

**Comment by atymic** [comment]
@Okipa can you make the above changes please? :)

**Comment by atymic** [comment]
Closing in favour of #35, thanks for the PR though :)

**Comment by Okipa** [comment]
Hey @atymic, sorry for the delay, I’m on vacations abroad and I’m trying to disconnect as much as I can.
Just saw you released the new tag with the Laravel 7 compatibility, thanks !

**Comment by atymic** [comment]
No worries :) 

All good, glad you had a good weekend!

**Pull request closed without merging**